src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -168,7 +169,10 @@ public void visit(NodeTraversal t, Node n, Node parent) {
 
       if (n.isGetProp() &&
           "module.exports".equals(n.getQualifiedName())) {
-        moduleExportRefs.add(n);
+        Var v = t.getScope().getVar(MODULE);
+        if (v == null) {
```

Comment:
Could you add a comment explaining the significance? E.g. `// Only rewrite the special top-level 'module' variable, but not local variables or parameters.`